app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError

```

Comment:
Do not duplicate imports, use `from app.errors import` once and place all required imports inside of `()` so that there would be no imports on the same line with opening and closing parenthesis